app/views/guests/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1><%= @guest.name %></h1>
+<h2>Rooms stayed in:</h2>
+<ul>
+<section id="rooms">
+  <% @rooms_stayed_in.each do |room| %>
+    <li>
+      Suite: <%= room.suite %>
+      Rate: <%= room.rate %>
+      Hotel: <%= room.hotel.name %>
+    </li>
+  <% end %>
+</ul>
+</section>
+
+<section id="add-room-form">
+  <%= form_with  url: "/guests/#{@guest.id}", local: true, method: :post do |form| %>
+    <%= form.label :room_id, "Room ID" %>
+    <%= form.text_field :room_id, name: "guest[room_id]"%>
+    <%= form.submit "Add Room" %>
+  <% end %>
+</section>
```

Comment:
Your route on line 16 could be a bit more restful since you're actually creating a GuestRoom. It could go to something like '/guest_rooms' and pass the guest id via a hidden field in the form. Or, it might even make more sense to have something like "/guests/:id/rooms"